backend/users/types.py

```diff
@@ -1,21 +1,58 @@
-from typing import List
+from typing import List, Optional
 
 import strawberry
 
 
+@strawberry.type
+class Image:
+    url: str
+
+
 @strawberry.type
 class MeUserType:
     id: strawberry.ID
     email: str
+    first_name: str
+    last_name: str
+    gender: str
+    open_to_recruiting: bool
+    open_to_newsletter: bool
+    date_birth: str
+    business_name: Optional[str]
+    fiscal_code: Optional[str]
+    vat_number: Optional[str]
+    recipient_code: Optional[str]
+    pec_address: Optional[str]
+    address: Optional[str]
+    country: str
+    phone_number: Optional[str]
+    image: Optional[Image]
```

Comment:
Image is defined two times here, and below with a resolver

Reply:
Mmm this could be a bug of strawberry? If leave only the resolver, I got an error when it tries to instantiate the `MeUserType` in `BaseUserMutation.transform`:

'__init__() got an unexpected keyword argument \'image\''

Reply:
Looks like a bug of Strawberry indeed, I will have a look